README.md

```diff
@@ -40,6 +40,18 @@ Run the following commands:
 1. `./gradlew createDB -PdbName=fineract_default`
 1. `./gradlew tomcatRunWAR`
 
+Instructions to download gradle wrapper for windows 10
+============
+go to https://gradle.org/
+click "install gradle" (https://gradle.org/install/)
+click "installing manually" (https://gradle.org/install/#manually)
+click "releases page" (https://gradle.org/releases/)
+go to version 2.10
+click complete;
+after the zip folder downloads extract the files into your C folder ( directory: "C:\gradle-2.10" );
+go to control panel -> system and security -> system -> advanced system settings -> environment variables; 
+under user variables click path -> edit -> new -> C:\gradle-2.10\bin then 
+click OK 
```

Comment:
I don't think this is necessary. Fineract platform ships with gradlew.bat by default which is which is an alternative to the standard gradlew for unix systems.
Running the gradle batch file takes care of installing gradle for you automatically.

Reply:
@xurror I was following the tutorial on this link https://cwiki.apache.org/confluence/display/FINERACT/Fineract-platform+Installation+on+Windows 
and it did not include the step of running the gradlew.bat file. I removed the path I created for gradle in my user environment and ran gradlew.bat file then went to gitbash and ran gradle clean dist and it built successfully so I guess what you suggested above works. 
 I have imagebot that I installed as part of the github education pack. It gets installed automatically in my repos which explains commits 
9e78318 and  3751ebc and I am sorry these commits are not relevant. Should probably deactivate imagebot :) I was only trying to edit README.md which I did in commit 
507e463 and then I added semi columns to make the instructions clearer in commit 4a1f521

Reply:
Right, so on windows you got to use gradlew.bat

I guess there is no need for this PR anymore.
So can you just close this?